cesm/driver/esm_time_mod.F90

```diff
@@ -150,13 +151,15 @@ subroutine esm_time_clockinit(ensemble_driver, instance_driver, logunit, maintas
                 open(newunit=unitn, file=restart_pfile, form='FORMATTED', status='old',iostat=ierr)
                 if (ierr < 0) then
                    rc = ESMF_FAILURE
+                   write(logunit,*) " error opening rpointer file "
                    call ESMF_LogWrite(trim(subname)//' ERROR rpointer file open returns error', &
                         ESMF_LOGMSG_ERROR, line=__LINE__, file=__FILE__)
                    return
                 end if
                 read(unitn,'(a)', iostat=ierr) restart_file
                 if (ierr < 0) then
                    rc = ESMF_FAILURE
+                   write(logunit,*) " error in read of rpointer file "
                    call ESMF_LogWrite(trim(subname)//' ERROR rpointer file read returns error', &
                         ESMF_LOGMSG_INFO, line=__LINE__, file=__FILE__)
```

Comment:
I realize this is pre-existing, but it seems like this ESMF_LOGMSG_INFO should be changed to ESMF_LOGMSG_ERROR (similar to the other ones above); do you agree?

Reply:
Oh, totally you are right. That's a simple and the right thing to do.

Reply:
Once  #529 is merged this should change to a shr_sys_abort call as is done there anyway. Which would resolve it by using the common subroutine.

And this is an example where cut and paste can get something wrong vs. using a common small subroutine.

Reply:
Ah, yes, I see this will be resolved by #532